__tests__/app.test.js

```diff
@@ -137,9 +137,141 @@ describe('GET /api/articles/:article_id', () => {
 })
 
 
+describe("Challenge 7 - POST /api/articles/:article_id/comments", () => {
+
+    test("status 201 - should respond with the posted comment when the correct properties are provided", () => {
+
+        const comment = {
+            username: "butter_bridge",
+            body: "Ey up lad fancy a jar?"
+        }
+
+        return request(app)
+        .post('/api/articles/3/comments')
+        .send(comment)
+        .expect(201)
+        .then(({body}) => {
+        
+        expect(body.comment).toHaveProperty('comment_id', 19)
+        expect(body.comment).toHaveProperty('body', 'Ey up lad fancy a jar?')
+        expect(body.comment).toHaveProperty('article_id', 3)
+        expect(body.comment).toHaveProperty('author', 'butter_bridge' )
+        expect(body.comment).toHaveProperty('votes', 0)
+        expect(body.comment).toHaveProperty('created_at', expect.any(String))
+
+        }) 
+    })
+
+    test("status 201 - should ignore unnecessary properties in the request body and respond with the posted comment with the correct properties", () => {
+
+        const comment = {
+            username: "butter_bridge",
+            body: "Ey up lad fancy a jar?",
+            ignorableProperty: "should ignore as its not a valid column in the table"
+        }
+
+        return request(app)
+        .post('/api/articles/3/comments')
+        .send(comment)
+        .expect(201)
+        .then(({body}) => {
+        
+        expect(body.comment).toHaveProperty('comment_id', 19)
+        expect(body.comment).toHaveProperty('body', 'Ey up lad fancy a jar?')
+        expect(body.comment).toHaveProperty('article_id', 3)
+        expect(body.comment).toHaveProperty('author', 'butter_bridge' )
+        expect(body.comment).toHaveProperty('votes', 0)
+        expect(body.comment).toHaveProperty('created_at', expect.any(String))
+
+        }) 
+    })
+    
 
 
 
 
+    test("sends a 400 when an invalid type request is sent" , () => {
```

Comment:
I think we can be clearer with the description here, it took me a minute to work out what this test is doing. Could we say this is for an invalid id